packages/jaeger-ui/src/utils/readJsonFile.test.js

```diff
@@ -72,4 +72,20 @@ describe('fileReader.readJsonFile', () => {
     const p = readJsonFile({ file });
     return expect(p).rejects.toMatchObject(expect.any(Error));
   });
+
+  it('loads JSON data with single resourceSpans array successfully', async () => {
```

Comment:
I don't think this test is needed, if you can process array of size>1 you can certainly process size=1